backend/kesaseteli/applications/models.py

```diff
@@ -115,7 +122,29 @@ class SummerVoucher(HistoricalModel, UUIDModel):
         verbose_name=_("unnumbered summer voucher reason"),
     )
 
-    employment_contract = models.FileField(
-        blank=True, verbose_name=_("employment contract")
+
+class Attachment(UUIDModel, TimeStampedModel):
+    """
+    created_at field from TimeStampedModel provides the upload timestamp
+    """
+
+    summer_voucher = models.ForeignKey(
+        SummerVoucher,
+        verbose_name=_("summer voucher"),
+        related_name="attachments",
+        on_delete=models.CASCADE,
     )
-    payslip = models.FileField(blank=True, verbose_name=_("payslip"))
+    attachment_type = models.CharField(
+        max_length=64,
+        verbose_name=_("attachment type in business rules"),
+        choices=AttachmentType.choices,
+    )
+    content_type = models.CharField(
+        max_length=100,
+        choices=ATTACHMENT_CONTENT_TYPE_CHOICES,
+        verbose_name=_("technical content type of the attachment"),
+    )
+    attachment_file = models.FileField(verbose_name=_("application attachment content"))
+
+    class Meta:
+        unique_together = [("summer_voucher", "attachment_type")]
```

Comment:
In Helsinki benefit I had to change attachment validation rules to allow multiple files of the same attachment type, because a single document might have multiple pages and the users would upload each page as a separate png file - don't know if the same applies here?

Reply:
I have to ask about this.

Reply:
Yea we will go with the same approach of allowing multiple attachments.